src/test/playwright/e2e/exercise/ExerciseImport.spec.ts

```diff
@@ -70,29 +73,72 @@ test.describe('Import exercises', () => {
+                await login(instructor, `/course-management/${secondCourse.id}/exercises`);
+                await courseManagementExercises.importQuizExercise();
+                await courseManagementExercises.clickImportExercise(multipleChoiceQuizExercise.id!);
 
-            await quizExerciseCreation.waitForFormToLoad();
-            await expect(page.locator('#field_title')).toHaveValue(quizExercise.title!);
-            await expect(page.locator('#quiz-duration-minutes')).toHaveValue(`${quizExercise.duration! / 60}`);
+                await quizExerciseCreation.waitForFormToLoad();
+                await expect(page.locator('#field_title')).toHaveValue(multipleChoiceQuizExercise.title!);
+                await expect(page.locator('#quiz-duration-minutes')).toHaveValue(`${multipleChoiceQuizExercise.duration! / 60}`);
 
-            await quizExerciseCreation.setReleaseDate(dayjs());
+                await quizExerciseCreation.setReleaseDate(dayjs());
 
-            const importResponse = await quizExerciseCreation.import();
-            const exercise: QuizExercise = await importResponse.json();
-            await courseManagementExercises.startQuiz(exercise.id!);
-            await login(studentOne, `/courses/${secondCourse.id}/exercises/${exercise.id}`);
-            await courseOverview.startExercise(exercise.id!);
-            await quizExerciseMultipleChoice.tickAnswerOption(exercise.id!, 0);
-            await quizExerciseMultipleChoice.tickAnswerOption(exercise.id!, 2);
-            const submitResponse = await quizExerciseMultipleChoice.submit();
-            const submission: QuizSubmission = await submitResponse.json();
-            expect(submission.submitted).toBe(true);
-            expect(submitResponse.status()).toBe(200);
-        });
+                const importResponse = await quizExerciseCreation.import();
+                const exercise: QuizExercise = await importResponse.json();
+                await courseManagementExercises.startQuiz(exercise.id!);
+                await login(studentOne, `/courses/${secondCourse.id}/exercises/${exercise.id}`);
+                await courseOverview.startExercise(exercise.id!);
+                await quizExerciseMultipleChoice.tickAnswerOption(exercise.id!, 0);
+                await quizExerciseMultipleChoice.tickAnswerOption(exercise.id!, 2);
+                const submitResponse = await quizExerciseMultipleChoice.submit();
+                const submission: QuizSubmission = await submitResponse.json();
+                expect(submission.submitted).toBe(true);
+                expect(submitResponse.status()).toBe(200);
+            },
+        );
+
+        test(
+            'Imports short answer quiz exercise',
+            { tag: '@fast' },
+            async ({ login, page, courseManagementExercises, quizExerciseCreation, courseOverview, quizExerciseShortAnswer }) => {
```

Comment:
Did you mean `quizExerciseShortAnswerQuiz`?
```suggestion
            async ({ login, page, courseManagementExercises, quizExerciseCreation, courseOverview, quizExerciseShortAnswerQuiz }) => {
```